Kittygram/ViewControllers/KittyDetails/KittyDetailsViewController.swift

```diff
@@ -7,18 +7,28 @@
 //
 
 import UIKit
+import RxSwift
+import RxCocoa
 
 class KittyDetailsViewController: UIViewController {
 
-    var kitty: Repository!
+    var kittyDetailsViewModel: KittyDetailsViewModel!
+    let disposeBag = DisposeBag()
     
     convenience init(kitty: Repository) {
         self.init()
-        
-        self.kitty = kitty
+        self.kittyDetailsViewModel = KittyDetailsViewModel(repository: kitty)
```

Comment:
Hmm, you don't need to create a new Coordinator, it could be added in existing one. What I meant is to create `viewModel` [here](https://github.com/sunshinejr/Kittygram/blob/mvvmc/Kittygram/Coordinator/DashboardCoordinator.swift#L24), and pass the `viewModel` in the `init`, instead of `repo`. The advantage is that `View` doesn't know anything about the `Model`, which is a core of `MVVM` architecture.   And we use Dependency Injection instead of hardcoded `viewModel` :)